tinytuya/Cloud.py

```diff
@@ -396,8 +396,11 @@ def _get_all_devices( self, uid=None, device_ids=None ):
     def getdevices(self, verbose=False, oldlist=[], include_map=False):
         """
         Return dictionary of all devices.
-        If verbose is true, return full Tuya device
-        details.
+
+        Args: 
+            verbose - Returns raw JSON data from Tuya Cloud
+            oldlist - List of devices from previous run
+            include_map - Include the DPS mapping in the device list

```

Comment:
I discovered I needed this better description of the args while adding the cloud sync button to the proxy. :-)